packages/velo-external-db/test/storage/data_provider.spec.js

```diff
@@ -53,6 +53,16 @@ describe(`Data API: ${currentDbImplementationName()}`, () => {
         })
     }
 
+    if (shouldRunOnlyOn(['Postgres', 'MySql'], currentDbImplementationName())) {
```

Comment:
add one test for case sensitive and one for case insensitive let's cover all the bases.